### DISCUSSION TITLE
idk how to get proper suggestions

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [florisboard/florisboard#1656] [Q&A]
Chk suggestion strip here it just adds 0 1 2 3 etc in the written word instead of correct word.
![Screenshot_2022-03-18-16-38-49-403_org thunderdog challegram](https://user-images.githubusercontent.com/38470944/159024565-bcc3b24c-bd82-4202-88a5-f23d826a2296.jpg)

Is there any settings or somethings which i should do to change that?

**Comment by jirafey** [comment]
i also have no idea if you are just using English switch to Anysoftkeyboard
korean does no have any suggestions as well

**Comment by HikaruIchijyo** [comment]
> i also have no idea if you are just using English switch to Anysoftkeyboard
> korean does no have any suggestions as well

Not a bug it is not  implemented in current version
